src/main/java/com/chariotsolutions/meetupnotifier/MeetupNotifierImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.chariotsolutions.meetupnotifier;
+
+
+import com.google.api.services.calendar.model.Event;
+
+import com.chariotsolutions.meetupnotifier.meetup.api.GoogleConverter;
+import com.chariotsolutions.meetupnotifier.meetup.api.MeetupQuery;
+import com.chariotsolutions.meetupnotifier.meetup.api.MeetupQueryExecutor;
+
+import com.chariotsolutions.meetupnotifier.meetup.api.messages.Results;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+@EnableScheduling
+public class MeetupNotifierImpl {
+
+  private static final Logger logger = LoggerFactory.getLogger(MeetupNotifierImpl.class);
+
+  private String[] groups;
+
+  public void run() throws IOException {
```

Comment:
I would rename the `run` method to `queryAndNotify`.  That method would then be exposed on the `MeetupNotifier` interface as well (we may expose other methods later, like just `query`, etc).  Eventually the calendar specific code can be moved into a separate strategy class ("a notification strategy") and `MeetupNotifier`/`MeetupNotifierImpl` will take in a list of strategies (which would be configured in `context.xml`).